.github/workflows/ci.yml

```diff
@@ -0,0 +1,493 @@
+name: GitHub Actions CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - develop
+      - githubactions*
+      - feature/**
+      - fix/**
+      - pr/**
+
+jobs:
+  posix:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "TOOLSET=gcc COMPILER=g++-4.4 CXXSTD=98,0x Job 0"
+            buildtype: "boost"
+            packages: "g++-4.4"
```

Comment:
Is there any real reason to spent public CI resources on outdated g++4.4?
The same is related to other outdated and unsupported compilers

Reply:
The file was automatically generated based on the existing .travis.yml which seems to still have g++4.4.    You are welcome to delete any outdated compilers.

Reply:
I don't think removing support for any compilers in Boost.Parameter was discussed or announced anywhere. I'd prefer as many compilers tested as possible.